bee-api/bee-rest-api/src/endpoints/routes/api/v2/submit_message.rs

```diff
@@ -78,30 +81,38 @@ pub(crate) async fn submit_message<B: StorageBackend>(
     rest_api_config: RestApiConfig,
     protocol_config: ProtocolConfig,
 ) -> Result<impl Reply, Rejection> {
-    let network_id_v = &value["networkId"];
+    let protocol_version_v = &value["protocolVersion"];
```

Comment:
what does `_v` stand for? Should this be obvious? If not, I'ld consider writing it out.

Reply:
It refers to the incoming JSON value, I wrote it out as you suggested 👍